index.html

```diff
@@ -346,7 +347,8 @@ <h2>Open Event Organizer App</h2>
         </div>
         
     </div>
-</section>
+ </section>
```

Comment:
Fix it.

Reply:
@arjunpandey2003  Why you added the end `</section>`, It is already there. Please remove it